Advice

Max-c-B opened the pull request:
Pull request to add my edits of my advice.html, the index.html and my advice CSS (on my advice branch) to the Skillcrush main branch.

redrambles commented:
Hey @Max-c-B !

This is a great  - we just need a few things before we can go ahead and merge this in.

Unfortunately we cannot accept images as part of student pull requests, as the repository would soon get too large. You are free to link to a version of the image that is hosted somewhere on the internet, though, of course! :) 

The other thing is that there are some merge conflict markers that remain in the index.html file ([see here](https://github.com/skillcrush/git-practice/pull/567/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R477)) - and some of the links have been duplicated (vosco, kat and libby).

1) Please delete the 'img' directory.
2) Please remove the conflict markers from the index.html file, as well as the three duplicate links.

Once these changes are complete, add them with `git add -A`, commit as per usual and push up your `advice` branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Cheers,
Ann :)

Max-c-B commented:
Hey Ann,

Thank you for your message and help. I've added the commits, so, I've deleted the img directory and I've removed the conflict marker and repeats. Now I think this is ready to be merged. Please could you advise me if it is not. 

Cheers
Max

twosavoie commented:
Hey @Max-c-B,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

xoxo,
Lisa
